### PR TITLE
feat(blade): add minHeight prop to CardProps

### PR DESCRIPTION
**Pull request by AritraLeo**
I've added the minHeight prop to CardProps and also to Card component in Card.tsx file. 
path - blade\packages\blade\src\components\Card\Card.tsx
issue - https://github.com/razorpay/blade/issues/1934

fixes https://github.com/razorpay/blade/issues/1934

### CODE REVIEW

File: packages/blade/src/components/Card/Card.tsx [hunk 1]
```diff
@@ -83,12 +83,17 @@ export type CardProps = {
    * Sets the height of the card
    */
   height?: BoxProps['height'];
+    /**
+    Stops UI breakage when height is 100% 
+   */
+  minHeight?: BoxProps['minHeight'];
```
Comment (iamakshatjain) [inline packages/blade/src/components/Card/Card.tsx:89]: 1. Formatting
2. Can update text to maybe just "sets minimum height of the card"
3. Can add min width also

File: packages/blade/src/components/Card/Card.tsx
Comment (iamakshatjain) [inline packages/blade/src/components/Card/Card.tsx:1]: You might need to pass these to CardRoot also, try running this on local also

File: packages/blade/src/components/Card/Card.tsx (hunk 1 above)
Comment (AritraLeo) [reply packages/blade/src/components/Card/Card.tsx:89]: Thanks !
I'll make the changes asap .

File: packages/blade/src/components/Card/Card.tsx
Comment (AritraLeo) [reply packages/blade/src/components/Card/Card.tsx:1]: Please have a look sir. If there is anything remaining please me know. 
Thanks !

**Review by iamakshatjain — APPROVED** [review]

**Comment by iamakshatjain** [comment]
@chaitanyadeorukhkar @kamleshchandnani @saurabhdaware please take a look at the PR

File: packages/blade/src/components/Card/Card.tsx [hunk 2]
```diff
@@ -83,12 +83,21 @@ export type CardProps = {
    * Sets the height of the card
    */
   height?: BoxProps['height'];
+    /**
+    Sets minimum height of the card
+   */
+  minHeight?: BoxProps['minHeight'];
+      /**
+    Sets minimum width of the card
+   */
```
Comment (anuraghazra) [inline packages/blade/src/components/Card/Card.tsx:92]: Please format the code same as how `width` is formatted. 

```suggestion
  /**
   * Sets minimum width of the card
   */
```

**Comment by anuraghazra** [comment]
@AritraLeo thanks for the PR. 

Can you add a story example for this in [here](https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Card/Card.stories.tsx)? 

You can add a new "SameHeightCard" story (refer to the examples)

File: packages/blade/src/components/Card/Card.tsx (hunk 2 above)
Comment (AritraLeo) [reply packages/blade/src/components/Card/Card.tsx:92]: Ok I'll do that and apologies for not doing the formatting properly this time !

**Comment by anuraghazra** [comment]
> yeah sure I can have a look but the task is actually not very clear to me. Can you please brief me a little bit more ?

You can look at the existing example [here](https://github.com/razorpay/blade/blob/e0caa7e223e13918ba02cfb6520c33d1fa98597c/packages/blade/src/components/Card/Card.stories.tsx#L371-L407). 

In this file you can add another example called `CardWithSameHeight`.

```jsx
const CardWithSameHeightExample: StoryFn<typeof Card> = (): React.ReactElement => {
  // Add 3 Cards with same heights by setting minHeight property to each of them
  return null;
};

export const CardWithSameHeight = CardWithSameHeightExample.bind({});
CardWithSameHeight.parameters = {
  controls: {
    disable: true,
  },
};
```

**Comment by anuraghazra** [comment]
FYI, tried this out locally, this doesn't work correctly. 

You also need to add minHeight prop to `CardSurface` because that's whats adds the background color/elevation etc.

**Comment by anuraghazra** [comment]
Merging this for now, we are releasing v11 soon and we want this change to be released in v10.

**Review by anuraghazra — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]
LGTM, thanks for contributing.

@kamleshchandnani @chaitanyadeorukhkar need your approval for merging.

**Comment by AritraLeo** [comment]
> FYI, tried this out locally, this doesn't work correctly.
> 
> You also need to add minHeight prop to `CardSurface` because that's whats adds the background color/elevation etc.

Apologies for the inconvenience I assure you it will not happen again I will be cautious.

**Review by kamleshchandnani — APPROVED** [review]
Thank you for your contribution 🎉

**Comment by anuraghazra** [comment]
> Apologies for the inconvenience I assure you it will not happen again I will be cautious.

No worries.

**Comment by AritraLeo** [comment]
> > Apologies for the inconvenience I assure you it will not happen again I will be cautious.
> 
> No worries.

I'll try to create a PR for - "SameHeightCard" story

**Pull request merged**